android/api.md

```diff
@@ -566,6 +566,278 @@ private static String MyOIDCAuthProvider(){
+
+You can also use Delta Sync functionality with GraphQL subscriptions, taking advantage of both only sending changes to the clients when they switch network connectivity but also when they are online. In this case you can pass a third query called the "Subscription Query" which is a standard GraphQL subscription statement. When the device is connected, these are processed as normal and the client API simply helps make setting up realtime data easy. However, when the device transitions from offline to online, to account for high velocity writes the client will execute the resubscription along with synchronization and message processing in the following order:
+
+1. Subscribe to any queries defined and store results in an incoming queue
+2. Run the appropriate query (If `refreshIntervalInSeconds` has elapsed, run the Base Query otherwise only run the Delta Query)
+3. Update the cache with results from the appropriate query
+4. Drain the mutation queue in serial
+
+Finally, you might have other queries which you wish to represent in your application other than the base cache hydration. For instance a `getItem(id:ID)` or other specific query. If your alternative query corresponds to items which are already in the normalized cache, you can point them at these cache entries with the `cacheUpdates` function which returns an array of queries and their variables. The DeltaSync client will then iterate through the items and populate a query entry for each item on your behalf. If you wish to use additional queries which don't correspond to items in your base query cache, you can always create another instance of the `client.sync()` process.
+
+**Usage**
+
+```java
+// Provides the ability to sync using a baseQuery, subscription, deltaQuery, and a refresh interval
+Cancelable handle = client.sync(baseQuery, baseQueryCallback, 
+			  subscription, subscriptionCallback, 
+			  deltaQuery, deltaQueryCallback, 
+			  20 * 60 );
+        
+//Stop DeltaSync
+handle.cancel();
+```
+
+**The method parameters**
+* *baseQuery* the base query to get the baseline state. (REQUIRED)
+* *baseQueryCallback* callback to handle the baseQuery results. (REQUIRED)
+* *subscription* subscription to get changes on the fly.
+* *subscriptionCallback* callback to handle the subscription messages. 
+* *deltaQuery* the catch-up query
+* *deltaQueryCallback* callback to handle the deltaQuery results. 
+* *baseRefreshIntervalInSeconds* time duration (specified in seconds) when the base query will be re-run to get an updated baseline state.
+* *returnValue* returns a `Cancelable` object that can be used later to cancel the sync operation by calling the `cancel()` method.
+
+Note that above only the `baseQuery` and `baseQueryCallback` are required parameters. You can call the API in different manners such as:
+
+```java
+client.sync(baseQuery, baseQueryCallback) //Performs sync only with base query
```

Comment:
add baseRefreshIntervalInSeconds as the last parameter in both of these options.